internal/admin/templates/bottom.html

```diff
@@ -1,10 +1,8 @@
 {{define "bottom"}}
 </div>
 
-  <script src="https://code.jquery.com/jquery-3.5.1.min.js"
```

Comment:
Bootstrap doesn't need jquery anymore and I verified that we're not using it anywhere.

Reply:
this is awesome. Should load faster too!